entities/src/main/java/com/flipkart/varadhi/entities/RetryTopic.java

```diff
@@ -0,0 +1,32 @@
+package com.flipkart.varadhi.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+/**
+ * TODO: generalize the retryTopic interaction. Currently, we assume that there is a retry topic per retry,
```

Comment:
I think not. 
I had a doubt, which I mentioned in comment. But on later thinking, it appears, that the interaction is actually part of producer. It will have a signature like `retryTopicProducer.produce(RetryMessage msg)`. 

The RetryTopic is just a data object specific to a particular impl, which will go into MetaStore.
https://github.com/flipkart-incubator/varadhi/issues/93 needs to be taken care of, if the data changes in any metaStoreEntity.

Reply:
Removed the TODO comment